### PR TITLE
updating questions

### PR DESCRIPTION
**Pull request by surajsahani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@
 | [Transact-SQL (T-SQL)](https://github.com/Ebazhanov/in-quiz-questions/blob/master/t-sql/t-sql-quiz.md) |  with answers  |
 | [WordPress](https://github.com/Ebazhanov/in-quiz-questions/blob/master/wordpress/wordpress-quiz.md) |  answers required  |
 | [XML](https://github.com/Ebazhanov/in-quiz-questions/blob/master/xml/xml-quiz.md) |  answers required |
+| [Android] |  answers required |
```
Comment (Ebazhanov) [inline README.md:49]: @surajsahani sorry I didn't mention but I already added a link to android(when I merged first time your first PR), please check it out. So could you please remove this line
```suggestion

```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Great job, just small comment to remove duplicate from Readme.md

File: README.md (hunk 1 above)
Comment (surajsahani) [reply README.md]: okay.

**Review by Ebazhanov — APPROVED** [review]
Good thanks!

**Pull request merged**